Util/MeshUtil.cs

```diff
@@ -248,12 +248,83 @@ public static string RenderableListToObjFile(List<IRenderable> meshes)
             return objString.ToString();
         }
 
+
+        public static string RenderableToPlyFile(IRenderable mesh)
+        {
+            int vertCount = mesh.Verts.Vertices.Length;
+            int triCount = mesh.Verts.Tris;
+
+            PlyBuilder plyString = new PlyBuilder();
+            plyString.WriteHeader(vertCount, triCount);
+
+            // Vertex Elements
+            foreach (var vert in mesh.Verts.Vertices)
+            {
+                plyString.BuildVertexElement(Vector3.TransformPosition(vert.Position, mesh.Transform.Matrix), vert.Normal, vert.TexCoord, vert.Color);
```

Comment:
Exceeds 120-character line limit as seen in rest of the codebase (sorry this wasn't in the README!)